src/Scrim.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Scrim from './Scrim';
+
+const setup = (propOverrides) => {
+  const props = Object.assign({
```

Comment:
since you are using the `spread` operator on line 10, maybe use the spread operator here as well?